blocks/lead-art-block/index.story.jsx

```diff
@@ -0,0 +1,506 @@
+import React from 'react';
+import LeadArt from './features/leadart/default';
+
+export default {
+  title: 'Blocks/Lead Art',
+  parameters: {
+    chromatic: {
+      viewports: [320, 1200],
+    },
+  },
+};
+
+const leadArtImage = {
+  _id: 'Y5HK2DX6WJHIBAV7VBMSIBFB6M',
+  type: 'image',
+  caption: 'ARLINGTON,VA - MARCH 2: Key Lime pie photographed for Voraciously in Arlington, Virginia on March 2, 2021 (Photo by Scott Suchman for The Washington Post; food styling by Lisa Cherkasky for The Washington Post)',
+  credits: {
+    affiliation: [{
+      name: 'food credit Lina',
+      type: 'author',
+    }],
+    by: [{
+      byline: 'Scott Suchman for The Washington Post',
+      name: 'Scott Suchman for The Washington Post',
+      type: 'author',
+    }],
+  },
+  subtitle: 'Key Lime Pie story in Voraciously',
+  url: 'https://cloudfront-us-east-1.images.arcpublishing.com/corecomponents/Y5HK2DX6WJHIBAV7VBMSIBFB6M.jpeg',
+  resized_params: {
```

Comment:
Does `LeadArt` require **all** these resized_params?

By only having the ones needed for the block it 
1. Allows us to see what is required for a given block
2. Reduces the overhead of parsing this file

Reply:
I honestly don't know which ones it needs.  I tried paring the options down to what was necessary, but this one got several levels deep into the image stuff and I stopped.  I can look into it.

Reply:
cut out a lot of these based on what is requested by the Image component